roles/set_openstack_containers/tasks/main.yml

```diff
@@ -105,14 +109,17 @@
   ansible.builtin.include_role:
     name: 'install_yamls_makes'
     tasks_from: 'make_wait'
+  when:
+    - operator_name != 'openstack'
+    - not cifmw_architecture_scenario is defined
```

Comment:
The fact we need to check against the presence of `cifmw_architecture_scenario` in this role is a big red flag. But it's out of the scope of your change.